everviz/data/load_csv/_get_data.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
+from webviz_config.common_cache import CACHE
+from webviz_config.webviz_store import webvizstore
+
+
+@CACHE.memoize(timeout=CACHE.TIMEOUT)
+@webvizstore
+def get_data(data_path) -> pd.DataFrame:
+    return pd.read_csv(data_path, delimiter=",")
```

Comment:
I _think_ we decided we could just go for the path to file for now, and change it with the everest_api when that is finished? In that case, we must ensure that the delimiter is correct (default everest is `;`)